recipes/thatool/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "thatool" %}
+{% set version = "1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 843304b7768932e6231f3f5ed5f8946b3ec3ebd3d4ac4a75bfba7ea85604739f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    # - thatool
+    # - thatool.filetool
+    # - thatool.free_energy_cal
+    # - thatool.modeling
+    # - thatool.parameter
+    # - thatool.utils
+
+about:
+  home: The package home page
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "This package contains the docs for several in-house codes to handle some specific tasks."
+  doc_url: 
+  dev_url: 
```

Comment:
fill these fields pls (include home above)

Reply:
done

Reply:
Can you either give `dev_url` a value or remove it?